.github/workflows/functional_test_v2.yaml

```diff
@@ -68,6 +68,12 @@ jobs:
       - name: Update dependencies
         run: |
           make dep-update
+      - name: Temporary generate new nodejs image - do not upload it
```

Comment:
Currently these images a pushed to quay.io in a separate step. This specific image is not that expensive to build, but, to facilitate upgrades that require changes to the test images we may want to consider saving the needed images as workflow artifacts instead of publishing them to quay.io

Reply:
That might help. That can be tracked separately.